includes/validation/class-amp-validation-manager.php

```diff
@@ -2413,8 +2421,31 @@ public static function enqueue_block_validation() {
 
 		wp_styles()->add_data( $slug, 'rtl', 'replace' );
 
+		$block_sources = BlockSources::is_needed() ? Services::get( 'dev_tools.block_sources' ) : null;
+
+		$plugin_registry = Services::get( 'plugin_registry' );
+
+		$plugin_names = array_map(
+			static function ( $plugin ) {
+				if ( isset( $plugin['Title'] ) ) {
+					return $plugin['Title'];
+				}
+
```

Comment:
Looks like that is indeed true. Updated in cd68effe2

Reply:
And this will be revisited with https://github.com/ampproject/amp-wp/issues/5697#issuecomment-749176913 wherein the WordPress.org plugin directory page URL will be used or else falling back to the `PluginURI` or else the `AuthorURI`.